{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2025-47711",
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "state": "PUBLISHED",
        "assignerShortName": "redhat",
        "dateReserved": "2025-05-07T21:08:45.449Z",
        "datePublished": "2025-06-09T06:03:47.063Z",
        "dateUpdated": "2025-06-09T18:19:57.680Z"
    },
    "containers": {
        "cna": {
            "title": "Nbdkit: nbdkit-server: off-by-one error when processing block status may lead to a denial of service",
            "metrics": [
                {
                    "other": {
                        "content": {
                            "value": "Moderate",
                            "namespace": "https://access.redhat.com/security/updates/classification/"
                        },
                        "type": "Red Hat severity rating"
                    }
                },
                {
                    "cvssV3_1": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "LOW",
                        "baseScore": 4.3,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
                        "version": "3.1"
                    },
                    "format": "CVSS"
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "There's a flaw in the nbdkit server when handling responses from its plugins regarding the status of data blocks. If a client makes a specific request for a very large data range, and a plugin responds with an even larger single block, the nbdkit server can encounter a critical internal error, leading to a denial-of-service."
                }
            ],
            "affected": [
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 10",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "nbdkit",
                    "defaultStatus": "affected",
                    "cpes": [
                        "cpe:/o:redhat:enterprise_linux:10"
                    ]
                },
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 7",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "nbdkit",
                    "defaultStatus": "unknown",
                    "cpes": [
                        "cpe:/o:redhat:enterprise_linux:7"
                    ]
                },
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 8",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "virt:rhel/nbdkit",
                    "defaultStatus": "unknown",
                    "cpes": [
                        "cpe:/o:redhat:enterprise_linux:8"
                    ]
                },
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 8 Advanced Virtualization",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "virt:8.2/nbdkit",
                    "defaultStatus": "unknown",
                    "cpes": [
                        "cpe:/a:redhat:advanced_virtualization:8::el8"
                    ]
                },
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 8 Advanced Virtualization",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "virt:av/nbdkit",
                    "defaultStatus": "unknown",
                    "cpes": [
                        "cpe:/a:redhat:advanced_virtualization:8::el8"
                    ]
                },
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 9",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "nbdkit",
                    "defaultStatus": "affected",
                    "cpes": [
                        "cpe:/o:redhat:enterprise_linux:9"
                    ]
                }
            ],
            "references": [
                {
                    "url": "https://access.redhat.com/security/cve/CVE-2025-47711",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_REDHAT"
                    ]
                },
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2365687",
                    "name": "RHBZ#2365687",
                    "tags": [
                        "issue-tracking",
                        "x_refsource_REDHAT"
                    ]
                }
            ],
            "datePublic": "2025-04-23T00:00:00.000Z",
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-193",
                            "description": "Off-by-one Error",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "x_redhatCweChain": "CWE-193: Off-by-one Error",
            "timeline": [
                {
                    "lang": "en",
                    "time": "2025-05-12T16:59:09.026000+00:00",
                    "value": "Reported to Red Hat."
                },
                {
                    "lang": "en",
                    "time": "2025-04-23T00:00:00+00:00",
                    "value": "Made public."
                }
            ],
            "providerMetadata": {
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat",
                "dateUpdated": "2025-06-09T06:03:47.063Z"
            }
        },
        "adp": [
            {
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2025-06-09T18:19:38.672467Z",
                                "id": "CVE-2025-47711",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2025-06-09T18:19:57.680Z"
                }
            }
        ]
    }
}